cupy/core/core.pyx

```diff
@@ -1383,6 +1383,9 @@ cdef class ndarray:
     def __hex__(self):
         return hex(self.get())
 
+    def __bytes__(self):
+        return self.tobytes()
```

Comment:
Does this work?

```suggestion
        return bytes(self.get())
```

Reply:
I haven't tried it yet. I will make the changes and test it by tomorrow.

Reply:
`bytes(self.get())` would probably work as per numpy as `self.get()` returns a numpy array. This would mean that corresponding tests for `numpy` and `cupy` will match, but not for `tobytes` and `bytes`

Reply:
Yeah that's ok I think. Getting `bytes` and `tobytes` to align sounds like a NumPy issue. Once they fix that, it will automatically be fixed in CuPy 🙂